pfio/MultiGroupServer.F90

```diff
@@ -820,7 +820,7 @@ subroutine start_back_writers(rc)
          file_size = file_size*4./1024./1024. ! 4-byte integer, unit is converted to MB
          speed = file_size/time
          lgr => logging%get_logger('MAPL.pfio')
-         call lgr%info(" Writing time: %f12.3 s, speed: %f12.3 MB/s, size: %f12.3 MB, oserver node: %i0~, file: %a", time, speed, file_size, this%node_rank, filename)
+         call lgr%info(" Writing time: %f9.3 s, speed: %f9.3 MB/s, size: %f9.3 MB, at server node: %i0~, of rank: %i0~, file: %a", time, speed, file_size, this%node_rank, this%innode_rank, filename)
```

Comment:
Can we condense this a bit:
```suggestion
         call lgr%info(" Writing time: %f9.3 s, speed: %f9.3 MB/s, size: %f9.3 MB, at server node: %i0~:%i0~, file: %a", time, speed, file_size, this%node_rank, this%innode_rank, filename)
```
We'll know what   137:4 means in this context.

I'm also thinking we'll regret `i0` as it will make the columns jagged, but at least it is at the end of the line.   And any particular width is either wasteful or wrong for big nodes in the future.

Reply:
In the interests of time,  I'll go ahead and commit my suggestion.